stream/cards.md

```diff
@@ -16,7 +16,30 @@ To facilitate users linking multiple cards, developers can add identifying key-v
 
 ### What Is Card Verification
 
-When linking a card, cardholders are required to complete a verification process to verify card ownership to ensure we protect them. The verification consists of the following steps:
+When linking a card, cardholders are required to complete one of the two available verification processes to verify card ownership to ensure we protect them.
+
+When linking a card, a property `verificationMethod` will identify what verification method that card is meant to follow - `3ds` or `microcharge`.
+
+### 3DS (3D Secure)
+
+3D Secure is a card verification method that is meant to enhance security measures for shoppers and vendors alike. This verification process consists of the following steps:
+
+1. The cardholder links their card to a Fidel API program using the Verified Enrollment SDK or via the Create Card and the Create Consent endpoints.
+2. A new screen prompting for additional information will be shown (i.e Cardholder's name and CVV/CVC)
```

Comment:
I think “a new screen” implies using our web SDK or it can imply our customer needing to present a new screen (which is not necessarily true, if the customer already knows that a Mastercard is being enrolled). I think this page should have the Fidel API as the first class citizen and refer to a Fidel API integration. So, an alternative formulation for this point would be: when calling the Create Consent endpoint, you would need to provide more cardholder details. Something like that.

Reply:
Yeah, that's a fair point, I'll try to change it accordingly